### ISSUE TITLE
follow_mouse = 2 does not work between float and tiled windows

### ISSUE THREAD

**Issue by kuba3351** [hyprwm/Hyprland#2548]
According to the documentation, when I set follow_mouse = 2 in the config file, mouse movement will not change my keyboard focus until I click on a window. But this does not work if I have a float Windows mixed with tiled Windows on a workspace.

When I move the mouse from float to tiled window, and from tiled window to float, the focus changes without clicking on a window. But If I move cursor between two float or two tiled windows, the focus does not change until I click the Window (works as expected).

**Comment by MightyPlaza** [comment]
Can confirm mouse follow_mouse setting doesn't affect floating windows.

**Comment by MightyPlaza** [comment]
Set the variable float_switch_override_focus = 0

**Comment by kuba3351** [comment]
Thank you, I missed it while reading documentation.

**Comment by ShayBox** [comment]
I also missed it, because the docs say only 1 and 2 are valid options.
